src/components/Datepicker/Datepicker.js

```diff
@@ -60,18 +60,23 @@ export class Datepicker extends React.Component {
       initTime,
       title,
       name,
-      showSingleHiddenInputValue
+      showSingleHiddenInputValue,
+      renderTime
```

Comment:
Just thinking about this - instead of `renderTime` I would call it `showTime`, because technically the 'time' part of the Datepicker is _always_ rendered (hidden by default). With this new prop we're adding a class to show it, rather than 'rendering' it 👍 

Happy to merge either way as this is minor!

Reply:
Is there a way to grab a variable name in all the files and change them all at once? @ttbarnes

Reply:
done